wordpress-mobile/WordPress-Android
# Add mark all as read functionality

Implement #20066

This is a feature about marking notifications as read in the current tab.

-----

## To Test:

1. Open the Jetpack app and switch to the notifications tab.
2. Scroll the top tab to `LIKES` and click it.
3. Click the overflow icon at the top-right corner.
4. Click the option: `Mark all as read`.
5. It should mark all notifications as read in `LIKES` tab.
6. Scroll the top tab to `COMMENTS` and click it.
7. Click the overflow icon at the top-right corner.
8. Click the option: `Mark all as read`.
9. It should mark all notifications as read in `COMMENTS` tab.
10. Done, thank you!

-----

## Regression Notes

1. Potential unintended areas of impact

    - Notification tab

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual

9. What automated tests I added (or what prevented me from doing so)

    - None

-----

## PR Submission Checklist:


-----

## Testing Checklist:
